distribution/lib/Standard/Visualization/0.0.0-dev/src/Scatter_Plot.enso

```diff
@@ -211,20 +222,22 @@ get_axes_field number_of_numeric =
 ## PRIVATE
 json_from_table : Table -> Vector Integer | Nothing -> Integer | Nothing -> Text
 json_from_table table bounds limit =
-    number_of_numeric_cols = table.numeric_column_count-2
+    x_value_type = table.x_value_type
+    x_is_date_or_time = x_value_type.is_date_or_time
+    number_of_numeric_cols = if x_is_date_or_time then table.numeric_column_count-1 else table.numeric_column_count-2
     fields_for_multiseries = Point_Data.all_fields + ((0.up_to number_of_numeric_cols).map idx-> Point_Data.Y_multi idx)
     fields_for_plot = if number_of_numeric_cols > 0 then fields_for_multiseries else Point_Data.all_fields
     data = table.point_data fields_for_plot |> bound_data bounds |> limit_data limit
     fields_for_axes = get_axes_field number_of_numeric_cols
     axes = table.axes fields_for_axes
     is_multi_series = number_of_numeric_cols > 0
-    JS_Object.from_pairs [[data_field, data], [axis_field, axes], ["is_multi_series", is_multi_series]] . to_json
+    JS_Object.from_pairs [[data_field, data], [axis_field, axes], ["is_multi_series", is_multi_series], ["x_value_type", x_value_type.to_display_text]] . to_json
```

Comment:
I believe the `to_display_text` here is to blame for nice human-readable representaiton. This is not what we want for this purpose, as it is likely to change over time. Using raw type name would be more stable.